library/src/main/java/com/bumptech/glide/manager/RequestManagerRetriever.java

```diff
@@ -60,6 +71,11 @@
    */
   private final Handler handler;
 
+  // Objects used to find Fragments and Activities containing views.
+  private final ArrayMap<View, Fragment> tempViewToSupportFragment = new ArrayMap<>();
+  private final ArrayMap<View, android.app.Fragment> tempViewToFragment = new ArrayMap<>();
```

Comment:
Will this work on version not having `android.app.Fragment`? My guess is yes, due to erasure.